### ISSUE TITLE
Crop Damage off

### ISSUE THREAD

**Issue by KnittingPaws** [drewhoener/JWSuggestions#54]
It would be really great if crops weren't damaged when jumping down one block, or jumping in general.

**Comment by johnnywoof** [comment]
Normal players are unable to trample crops. Or did I miss something?

**Comment by Spazgazum** [comment]
If you have permissions for the land, you can. Happens all the time on accident. 

+1 support on not trampling my own crops.

**Comment by KnittingPaws** [comment]
ummm...i end up trampling mine all the time :( is there a command or flag? cause i can't find it

**Comment by hoogkerker** [comment]
she means the minecraft physics in general.
if you are like flying above farmland and land you will trample crops, except if area is protected, and she asks if that could be changed.

**Comment by KnittingPaws** [comment]
Yeah. So in my own fields I end up trampling the crops just from walking up and down the levels of my farm when i'm planting or harvesting. Or when a hostile mob gets on my farm and hits me, when I land I trample my crops too :(. So i'm wondering if there's a way to turn off crop trampling all together.

**Comment by drewhoener** [comment]
I could probably add a flag for that

**Comment by Spazgazum** [comment]
Awesome. Please and thank you : )

**Comment by johnnywoof** [comment]
It should just be globally disabled like in 2.0.
@drewhoener

**Comment by drewhoener** [comment]
I still like the idea of a flag better

**Comment by johnnywoof** [comment]
I still like the idea of it being globally disabled.

**Comment by drewhoener** [comment]
Well maybe there are times when I want to trample my own crops. Sponge flag it is!

**Comment by 5d100** [comment]
the only problem is that sponge flags don't apply to the people in /wp, so even if you turn it off, it won't do any good to the people who turn it off.

**Comment by drewhoener** [comment]
@5d100 @johnnywoof The thing is, the more things we just globally disable across the board, the less freedom players can have, and it reduces the amount of things that can happen in the game due to chance. Say your cow gets loose and walks over all your precious wheat. Sure that's a bit unfortunate, but that's part of the game and I don't see a reason to turn it off everywhere just because someone finds it a bit irksome. If you want to disable players from trampling crops, that's fine. But it's not happening everywhere
